projects/storefrontlib/cms-components/product/product-list/product-list-item/product-list-item.component.html

```diff
@@ -19,12 +19,12 @@
     </a>
   </div>
   <div class="col-12 col-md-8">
-    <a
-      [routerLink]="{ cxRoute: 'product', params: product } | cxUrl"
-      class="cx-product-name"
-    >
-      <h2 [innerHtml]="product.nameHtml"></h2>
-    </a>
+    <h2 class="cx-product-name">
```

Comment:
This will also be a breaking DOM change similar to this one:
https://github.com/SAP/spartacus/pull/18268#discussion_r1432623453